livekit-rtc/livekit/rtc/room.py

```diff
@@ -240,10 +240,10 @@ async def _listen_task(self) -> None:
                         event.room_event,
                     )
 
-            # wait for the subscribers to process the event
-            # before processing the next one
-            self._room_queue.put_nowait(event)
-            await self._room_queue.join()
+                # wait for the subscribers to process the event
+                # before processing the next one
+                self._room_queue.put_nowait(event)
```

Comment:
so we will wait for each event now and join?

Reply:
removed